tests/workloads/slurm_container/test_slurm_container.py

```
@@ -0,0 +1,81 @@
+
+    @pytest.mark.parametrize(
+        ("exit_code", "is_successful"),
+        (
+            ("0:0", True),
+            ("0", True),
+            ("0:15", True),
+            ("1:0", False),
+            ("15:0", False),
+            ("42:0", False),
+            ("137:0", False),
+        ),
```

Ruff (0.15.20)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)